timify-api/app.js

```diff
@@ -15,7 +15,8 @@ app.use(morgan('combined'))
 const endpointURL = "https://api.taddy.org"
 
 async function taddyGraphqlRequest(query, variables) {
-  const response = await axios({
+  try {
+    const response = await axios({
```

Comment:
Please format this try catch block and the 'headers' field has extra space between field name and bracket.